### PR TITLE
Add base_url in /etc/linaro/ kernelci-backend.cfg

### PR DESCRIPTION
**Pull request by ana**
This update should be done to complement https://github.com/kernelci/kernelci-backend/pull/81

### CODE REVIEW

File: roles/install-app/templates/kernelci-backend.cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 master_key = '{{ master_key }}'
-storage_url = 'http://storage.kernelci.org/'
+storage_url = 'https://storage.kernelci.org/'
```
Comment (gctucker) [inline roles/install-app/templates/kernelci-backend.cfg:2]: Are you sure about the trailing `/` here?  Should it not be `https://storage.kernelci.org` instead?

Comment (gctucker) [reply roles/install-app/templates/kernelci-backend.cfg:2]: Also the commit summary "use https" is a bit vague.

**Comment by gctucker** [comment]
The "kernelci-backend.cfg: Add base_url" commit message is missing a blank line before "Signed-off-by".

File: roles/install-app/templates/kernelci-backend.cfg (hunk 1 above)
Comment (ana) [reply roles/install-app/templates/kernelci-backend.cfg:2]: Links currently work with the the trailing '/' there so I wouldn't recommend to remove it without checking for possible breakage.

**Pull request merged**